docs/multicluster/user-guide.md

```diff
@@ -484,20 +516,27 @@ default-nginx-service     99s
 ```
 
 When there is any new change on the exported Service, the imported multi-cluster
-Service resources will be updated accordingly. But when multiple member clusters
-exported the same Service with conflicted definitions, only the first export
-will be replicated to other clusters, and new updates on the Service will be
-ingored, until user fixes the conflicts. For example, after a member cluster
-exported a Service: `default/nginx` with TCP Port `80`, other clusters can only
-export the same Service with the same Ports definition including Port names.
+Service resources will be updated accordingly. Multiple member clusters can
+export the same Service (with the same name and Namespace). In this case, the
+imported Service in a member cluster will include endpoints from all the export
+clusters, and the Service requests will be load-balanced to all these clusters.
```

Comment:
Should we highlight there is no difference between local member cluster and remote clusters' Services if local one exports the same Service as well?

Reply:
Sorry. Could you explain what you mean?

Reply:
I mean it randomly pick one Endpoint for multi-cluster Service to route the traffic, so we can't guarantee local Pod's request will go to local exported Service's Endpoint.

Reply:
Ok. Added that.